FluEgggui.m

```diff
@@ -374,53 +374,33 @@
             %% ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
             if strcmp(specie,'Silver')%if specie=='Silver'
                 %% STD
-                if time(tt)/3600<4
-                    DiamStd = 0.9868;%mm
-                else
-                    DiamStd = 0.3512;%mm
-                end
-                if time(tt)/3600<1
-                    RhoeStd = 8.2231;%Kg/m^3 at 22C
-                else
-                    RhoeStd = 0.7780;%Kg/m^3 at 22C
-                end
-                %%
+                    %% Diameter
+                    DiamStd = 0.2631.*exp(-time/22410)+0.3073;
+                	%% Density
+                    RhoeStd = 22.4.*exp(-time/1894)+0.4103;
```

Comment:
Lori, can you please verify the equation of Density STD?? it does not match with the ppt